src/components/RightSidebar.js

```diff
@@ -72,6 +73,14 @@ const RightSidebar = React.createClass({
       { header: L10N.getStr("scopes.header"),
         component: Scopes }
     ];
+
+    if (isEnabled("eventListeners")) {
+      items.push({
+        header: "EventListeners",
```

Comment:
Here we'd want `Event Listeners` instead no (probably an L10N string would be better.)